test/Feature/MultipleSchemaSpec.hs

```diff
@@ -0,0 +1,196 @@
+        [json|[
+          {"id":1,"value":"value1"},
+          {"id":2,"value":"value2"}
+        ]|]
+        {
+          matchStatus = 200
+        , matchHeaders = [matchContentTypeJson]
+        }
+
+    it "read table from other schema(v2)" $
+      request methodGet "/table" [("Accept-Version", "v2")] "" `shouldRespondWith`
+        [json|[
+          {"id":1,"value":"value3"},
+          {"id":2,"value":"value4"}
+        ]|]
+        {
+          matchStatus = 200
+        , matchHeaders = [matchContentTypeJson]
+        }
+
+    it "read another_table from other schema(v2)" $
+      request methodGet "/another_table" [("Accept-Version", "v2")] "" `shouldRespondWith`
+        [json|[
+          {"id":1,"value":"value5"},
+          {"id":2,"value":"value6"}
+        ]|]
+        {
+          matchStatus = 200
+        , matchHeaders = [matchContentTypeJson]
+        }
+
+    it "read table from unkown schema" $
+      request methodGet "/table" [("Accept-Version", "unkown")] "" `shouldRespondWith`
+        [json|{"message":"The schema must be one of the following: v1, v2"}|]
+        {
+          matchStatus = 406
+        , matchHeaders = []
+        }
+
+    it "read table definition from default schema (v1)" $ do
+        r <- simpleBody <$> request methodGet "/" [] ""
+
+        let def = r ^? key "definitions" . key "table"
+
+        liftIO $
+
+          def `shouldBe` Just
+              [aesonQQ|
+                {
+                  "properties" : {
+                    "id" : {
+                      "description" : "Note:\nThis is a Primary Key.<pk/>",
+                      "format" : "integer",
+                      "type" : "integer"
+                    },
+                    "value" : {
+                      "format" : "text",
+                      "type" : "string"
+                    }
+                  },
+                  "required" : [
+                    "id",
+                    "value"
+                  ],
+                  "type" : "object"
+                }
+              |]
+
+    it "read table definition from default schema (v1) after explicitly passing it in the header" $ do
+        r <- simpleBody <$> request methodGet "/" [("Accept-Version", "v1")] ""
+
+        let def = r ^? key "definitions" . key "table"
+
+        liftIO $
+
+          def `shouldBe` Just
+              [aesonQQ|
+                {
+                  "properties" : {
+                    "id" : {
+                      "description" : "Note:\nThis is a Primary Key.<pk/>",
+                      "format" : "integer",
+                      "type" : "integer"
+                    },
+                    "value" : {
+                      "format" : "text",
+                      "type" : "string"
+                    }
+                  },
+                  "required" : [
+                    "id",
+                    "value"
+                  ],
```

Comment:
Ditto. After correcting all of these the test suite should be green.

Reply:
In order for the tests to turn green I need to additionally remove the "value" object under "properties", but is the output I get when calling the endpoint
![curl_output](https://user-images.githubusercontent.com/5838197/75846692-614fd980-5ddd-11ea-8554-ad5566bf743b.png)